examples/ListenerAgent/listener/agent.py

```diff
@@ -107,7 +107,7 @@ def onstart(self, sender, **kwargs):
         query = Query(self.core)
         _log.info('query: %r', query.query('serverkey').get())
 
-    @PubSub.subscribe('pubsub', '')
```

Comment:
Why changing behavior

Reply:
This is an example agent so it is good to include the all_platforms feature also in it. An example code to subscribe to all platforms..